.travis.yml

```diff
@@ -4,17 +4,34 @@ sudo: false
 cache:
   - pip
 
+before_cache:
+  - rm ~/.cache/pip/log/debug.log
+
 language: python
 python:
   - "2.7"
   - "3.5"
 
 install:
-  - pip install -r requirements.txt
   - pip install tox-travis
 
 script:
-  # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
-  - ansible-playbook --syntax-check playbooks/byo/config.yml
   - tox
-  - cd utils && tox
+
+notifications:
```

Comment:
Is this indeed intended or was just part of debugging coveralls?
I'm ok with it, just double checking ;)